examples/apps/ai-collab/src/app/tasks-list/layout.tsx

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+"use client";
+
+import { Box } from "@mui/material";
+// NextJS requires reaching out to internal modules.
+/* eslint-disable import/no-internal-modules */
```

Comment:
Worth adding a project-wide disable specifically for the nextjs imports?

Reply:
Good point, the setup was already there with some allows, so I added nextjs stuff to it.